src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfRestartService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Should these tests still exist? I really don't like the idea of killing IIS to test things

Not blocking